src/Microsoft.FeatureManagement.AspNetCore/TargetingHttpContextMiddleware.cs

```diff
@@ -50,14 +52,15 @@ public async Task InvokeAsync(HttpContext context, ITargetingContextAccessor tar
 
             if (targetingContext != null)
             {
-                context.Items[TargetingIdKey] = targetingContext.UserId;
+                var activityFeature = httpContext.Features.Get<IHttpActivityFeature>();
+                activityFeature.Activity.AddBaggage(TargetingIdKey, targetingContext.UserId);
```

Comment:
Can `Activity` be null?

Reply:
I see conflicting examples around this- so I'll add a null check to be safe. I suppose this will become:
```csharp
activityFeature?.Activity?.AddBaggage(TargetingIdKey, targetingContext.UserId);
```